cmd/node/main.go

```diff
@@ -569,13 +569,14 @@ func getPrometheusJoinURL(serversConfigurationFileName string) (string, error) {
 	if err != nil {
 		return "", err
 	}
-	joinURL := serversConfig.Prometheus.PrometheusJoinURL
-	statusURL := strings.Replace(joinURL, "/join", "/status", 1)
-	_, err = http.Get(statusURL)
-	if err != nil {
+	baseURL := serversConfig.Prometheus.PrometheusBaseURL
+	statusURL := baseURL + serversConfig.Prometheus.StatusRoute
+	resp, err := http.Get(statusURL)
+	if err != nil || resp.StatusCode == http.StatusNotFound {
```

Comment:
Maybe it's better to have two different conditions here. If you will get 404 `err` can be nil

Reply:
fixed